README.md

```diff
@@ -33,22 +33,26 @@ Otherwise, you can follow these steps:
    ```sh
    git clone git@github.com:NewShadesDAO/api.git
    ```
-2. Install packages
+2. Install poetry (Python dependency management tooling) via pipx
    ```sh
-   pip install -r requirements.txt
+   pip install pipx && pipx install poetry
    ```
-3. Setup local .env file
+3. Install packages
    ```sh
-   copy .env.template .env
+   poetry install
```

Comment:
if I'm not using `virtualenv` or `pyenv` (which I was before), and I run `poetry install`, will it create a venv by default and then use it for all `poetry run` stuff, is that it?

Reply:
@pedropregueiro Yes, exactly. The default is that `poetry` handles the virtualenvs on its own (unless you've already activated a virtualenv on your own). Virtualenvs via `poetry` can however be completely disabled with:
```sh
poetry config virtualenvs.create false 
poetry config virtualenvs.in-project false
```